homeassistant/components/citybikes/manifest.json

```diff
@@ -1,7 +1,10 @@
 {
   "domain": "citybikes",
   "name": "CityBikes",
+  "version" : "1.0.0",
```

Comment:
The `version` is only required for custom components. For core integrations, this should be omitted.

Reply:
It is a bit stronger, it is not allowed.